examples/peripheral/uart/src/main.rs

```diff
@@ -0,0 +1,37 @@
+//! Example of sending a software break signal from a UART in
+//! Blocking mode.
+//!
+//! The following wiring is assumed:
+//! - TX => GPIO17
+//% CHIPS: esp32
+//% FEATURES: esp-hal/unstable
```

Comment:
I'm not sure if these directives work with example projects, but they are definitely unnecessary.

Reply:
Fixed in [8426ce4](https://github.com/esp-rs/esp-hal/pull/4284/commits/8426ce4c529a686ca3820aec6fe559eb53646ea1)